mne/stats/regression.py

```diff
@@ -134,3 +143,208 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
+
+def linear_regression_raw(raw, events, event_id=None,
+                          tmin=-.1, tmax=1,
+                          covariates=None,
+                          reject=True, tstep=1.,
```

Comment:
Yeah I see that's what it takes -- my question is, is this behavior and set of options consistent with what we do with the `reject` parameter in other functions that take a `reject` argument? It will take some looking around to figure that out I think.